source/versions/history.rst

```diff
@@ -7,55 +7,55 @@ Package History
 Major Features and Improvements
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Implemented support for deep neural networks and [Huggingface](https://huggingface.co/models) models in our current NLP pipeline.
+- Implemented support for deep neural networks and `Huggingface <https://huggingface.co/models>`_ models in our current NLP pipeline.
 
 - Introduce active learning support for entity recognition.
 
-- Improved the performance and speed of the [Tokenization pipeline](https://github.com/cisco/mindmeld/pull/398).
+- Improved the performance and speed of the `Tokenization pipeline <https://github.com/cisco/mindmeld/pull/398>`_.
 
 - Updated Duckling dependency with the latest upstream changes.
 
 Breaking Changes
 ^^^^^^^^^^^^^^^^
 
-- Fixed a [bug](https://github.com/cisco/mindmeld/pull/405) with the EntityLabelEncoder by refactoring the System Entity Recognizer attribute.
-- Fixed a [bug](https://github.com/cisco/mindmeld/pull/387) that ensures all model cache files are saved properly during an incremental build.
+- Fixed a `bug <https://github.com/cisco/mindmeld/pull/405>`__ with the EntityLabelEncoder by refactoring the System Entity Recognizer attribute.
+- Fixed a `bug <https://github.com/cisco/mindmeld/pull/387>`__ that ensures all model cache files are saved properly during an incremental build.
 
 After these 2 fixes, older pickled models that refer to the previous versions will not load, so delete the ``.generated`` folder in application folder, and rebuild all classifiers.
 
 
 Bug Fixes and Other Changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Fixed an [issue](https://github.com/cisco/mindmeld/pull/400) with the build of docker containers.
-- Set [default tokenizer](https://github.com/cisco/mindmeld/pull/397) for all languages to Whitespace.
-- Refactor how the resource loader is [initialized in NativeQA](https://github.com/cisco/mindmeld/pull/391)
-- Fix a [bug](https://github.com/cisco/mindmeld/pull/390) related to updating the MM version.
-- Remove a redundant warning in [TextPreparationPipeline creation](https://github.com/cisco/mindmeld/pull/385).
+- Fixed an `issue <https://github.com/cisco/mindmeld/pull/400>`__ with the build of docker containers.
```

Comment:
Definitely not a big issue, and it looks like things render perfectly fine either way, but there are some differences between using a single underscore _ and two of them __ in some of these links in this file

Reply:
The double underscore just tells doc compiler not to treat these as named links, to squash some warnings.